plugins/inputs/opentelemetry/sample.conf

```
@@ -7,6 +7,9 @@
   ## Override the default (5s) new connection timeout
   # timeout = "5s"
 
+  ## gRPC Maximum Message Size
+  # max_msg_size = "4MB"
```

Maybe
```suggestion
  # max_message_size = "4MB"
```
Even though I hate typing a lot I think using abbreviations is not the best for everybody. ;-)

---

Yeah, I decided to be consistent with the other gRPC plugins that have this same option :\